stdlib/src/builtin/file.mojo

```diff
@@ -33,8 +33,7 @@ with open("my_file.txt", "r") as f:
 
 from os import PathLike
 from sys import external_call
-
-from memory import AddressSpace, DTypePointer, Pointer
+from memory.unsafe import AddressSpace, DTypePointer, Pointer
```

Comment:
**Suggestion** Do you mind not making this change?  We'd prefer `from memory <blah>` rather than `from memory.unsafe <blah>`.  See https://github.com/modularml/mojo/commit/63d5cb0e5fd1b90fb4da5c8e4f5974fbc04c8633